src/test/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTriggerTest.java

```diff
@@ -1149,14 +1154,6 @@ public void testGerritEventManualEvent() {
         trigger.createListener().gerritEvent(event);
 
         verify(listener).onTriggered(same(project), same(event));
-
-        verify(project).scheduleBuild2(
```

Comment:
you are removing a verification step here, so the test isn't testing the same thing as before anymore.